.github/workflows/cargo-contract-build.yml

```diff
@@ -27,5 +27,5 @@ jobs:
           && sudo cp binaryen-version_110/include/* /usr/include \
           && sudo cp binaryen-version_110/lib/* /lib/ \
           && rm -rf binaryen-version_110"
-      - name: Run cargo contract build for executor contracts
-        run: cd contracts/index_executor && cargo contract build
+      - name: Build executor contract
+        run: cd contracts/index_executor && cargo contract build --release
```

Comment:
why release? is there any scenario where plain "cargo contract build" succeeds but "cargo contract build --release" fails?

Reply:
If we don't add --release, cargo contract will add debug feature when we build the contract, which needs us to create the feature explicitly

Reply:
the purpose of this CI task is not to release a binary but to test if the code satisfies `cargo contract` requirement, adding `--release` does not make any sense according to this purpose, and wastes more time building a thing that is not going to be used for each commit you add the PR

If you want to release a binary, do it separately, use `git tag` to label the version, and build that binary on a specific condition, not like this. But even though this is doable, it doesn't mean we should do it, who's going to use that binary? It's just us, this is not like some other popular repos with thousands of people wanting for executable release.

That said, if we truly want to release versioned binary, we should discuss about it.